actors/market/src/v10/state.rs

```diff
@@ -1075,6 +1147,33 @@ impl State {
 
         self.unlock_balance(store, addr, amount, lock_reason)
     }
+
+    /// Verify that a given set of storage deals is valid for a sector currently being PreCommitted
+    pub fn verify_deals_for_activation<BS>(
+        &self,
+        store: &BS,
+        deal_ids: Vec<DealID>,
+        addr: &Address,
+        sector_exp: i64,
```

Comment:
I was thinking of this parameter order:
https://github.com/filecoin-project/lotus/blob/aa76a450862559c16bb187968e20b6aaf51d247e/node/impl/full/state.go#L1463

Rationale: have both clients call the same functionality similarly (principle of most minor surprise).